### PR TITLE
Json server2

### PR DESCRIPTION
**Pull request by Craftsman8**
# New features

- Remove unused files
- Add router links
- Add BlogDetails to contain the blog list
- Add form page
- Add submit button
- Add styles to the form page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-        </div>
-      <Home/>
-      <Navbar />
-      <BlogDetails />
-      <Sidebar/>
+     </Routes>
+      
       <Toggle />
-      <Features/>
       <Components />
```
Comment (Bestbynature) [inline src/App.js:27]: @Craftsman8 
These three components are outside the Routes tag which means they will always display in the DOM no matter the route you are in. I believe that is not what you want.

For example, you want the Components component to be loaded when the user clicks home or when the app starts initially. This is why you should remove it from here and nest it inside the Home component itself.

**Review by Bestbynature — CHANGES_REQUESTED** [review]
Almost There GIF

Hello @Craftsman8, you did a wonderful job with your colour choices and the overall design. Kindly fix the issue pointed out and you will be on your way to merge to dev.

File: src/App.js (hunk 1 above)
Comment (Craftsman8) [reply src/App.js:27]: oh...yes

Comment (Craftsman8) [reply src/App.js:27]: oh...yes, thank you sir. I wanted to ask before
I've tried to get rid of it but couldn't

**Pull request merged**